tests/test_util.py

```diff
@@ -222,12 +222,19 @@ async def asyncSetUp(self):
     async def asyncTearDown(self):
         await self.http.close()
 
+    async def test_correct_content_type(self):
+        await get_extra_data_info_from_url(
+            url=f"https://ftp.gnu.org/gnu/gzip/gzip-1.12.tar.gz",
+            session=self.http,
+            content_type_deny=[re.compile(f"^{self._CONTENT_TYPE}$")],
```

Comment:
I don't think the class-level constant adds anything here – why not inline it?

```suggestion
            content_type_deny=[re.compile(r"^application/x-fedc-test$")],
```